modules/security/pages/password-policy.adoc

```diff
@@ -20,6 +21,14 @@ There are also rules for how long a password may be used before it must be chang
 * How long the password remains valid after it has been updated
 * The minimum number of unique passwords required before a previously used password can be reused
 
+== Password expiration warning
+When the password is set to expire within the next 7 days, users will receive a warning during login. This feature ensures users have enough time to update their passwords and avoid being locked out of the system.
+
+*For example:*
```

Comment:
@Tushar-TG-14 Can you ask the developer to provide an example of logging in and how the warning message will look.